docs/source/introduction/ext_cfg_util.rst

```diff
@@ -0,0 +1,73 @@
+The purpose of the VISA external configuration utility
+==============================================================================
+
+Managing system VISA resources -
+------------------------------------------------------------------------------
+The three main VISA distributions all come with a utility to manage VISA
```

Comment:
NI, R&S, Keysight I assume. Does Tek miss one ?

Reply:
NI, R&S, Keysight are "Sponsor" members of IVI foundation, but I see Tek and Keithley (a Tek company) are "General" members. `https://www.ivifoundation.org/membership/current_members.aspx`  I initially got the impression that Tek had abandoned providing/supporting TekVISA distributions.  After a more thorough search on the Tek website I did find: `https://www.tek.com/en/support/software/driver/tekvisa-connectivity-software-v420`

I personally think the fewer VISA distributions PyVISA mentions the easier it is to support users.  I would recommend omitting mention of TekVISA unless someone from Tek asks for it to be added.

Please let me know if you would like me to add TekVISA.